docs/_layouts/docs.html

```diff
@@ -228,6 +228,11 @@ <h5>Troubleshooting</h5>
                    Stuck App or Pod Deployments
                 </a>
             </li>
+            <li>
+                <a href="{{ site.baseurl }}/docs/backoff.html">
+                    No new instances are starting
```

Comment:
this is troubleshooting... my concern with putting a link specifically for backoff delay for "No new instances are starting" is that there could be multiple reasons why no new instances are starting and this is one of the reasons.   Wouldn't it make more sense to have a list for No new instances are starting of reasons why with a link to each.. which includes this one?